src/main/cms/cms_menu_failsafe.c

```diff
@@ -0,0 +1,90 @@
+/*
+ * This file is part of Cleanflight and Betaflight.
+ *
+ * Cleanflight and Betaflight are free software. You can redistribute
+ * this software and/or modify this software under the terms of the
+ * GNU General Public License as published by the Free Software
+ * Foundation, either version 3 of the License, or (at your option)
+ * any later version.
+ *
+ * Cleanflight and Betaflight are distributed in the hope that they
+ * will be useful, but WITHOUT ANY WARRANTY; without even the implied
+ * warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this software.
+ *
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include <stdbool.h>
+#include <stdint.h>
+#include <string.h>
+#include <ctype.h>
+
+#include "platform.h"
+
+#if (defined(USE_CMS) && defined(USE_CMS_FAILSAFE_MENU))
```

Comment:
A better way to do this would be to only check for `USE_CMS_FAILSAFE_MENU`, and undefine `USE_CMS_FAILSAFE_MENU` in `common_post.h` if `USE_CMS` isn't defined.

Reply:
Done.